robottelo/cli/hostcollection.py

```diff
@@ -101,6 +106,13 @@ def hosts(cls, options=None):
             cls._construct_command(options), output_format='csv')
 
     @classmethod
+    def list(cls, options=None, per_page=True):
+        """List host collections"""
+        if options:
+            cls.transform_ids(options)
```

Comment:
see my comment about refactor the transform_ids function above, that allow to call it without checking